pilot/pkg/proxy/envoy/v1/config.go

```diff
@@ -915,6 +915,11 @@ func appendPortToDomains(domains []string, port int) []string {
 	return domainsWithPorts
 }
 
+// TruncateClusterName to a fixed size string using SHA if necessary
+func TruncateClusterName(name string) string {
+	return truncateClusterName(name)
+}
+
 func truncateClusterName(name string) string {
```

Comment:
Just make this one public; no reason to add a public func that directly calls a private func of the same name, parameters, and return type.

Reply:
Trying to minimize any change to v1 per @costinm 's requests.

Reply:
This is a trivial change, just make it.